sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -65,7 +65,8 @@ def raise_with_traceback(exception, *args, **kwargs):
     message = kwargs.pop("message", "")
     exc_type, exc_value, exc_traceback = sys.exc_info()
     # If not called inside a "except", exc_type will be None. Assume it will not happen
-    exc_msg = "{}, {}: {}".format(message, exc_type.__name__, exc_value)  # type: ignore
+    if exc_type is not None:
```

Comment:
Think deeper here: what does it mean for this function that ext_type is None? Is this a valid state that we want to workaround it, or is it an invalid state? What this function is here for.